### PR TITLE
Python Code Contributions Guidelines (CADC-13191)

### PR DESCRIPTION
**Pull request by andamian**

### CODE REVIEW

File: cadcutils/CONTRIBUTIONS.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+* `README.md` - description of the project
+* `LICENCE` - the license file
+* `tox.ini` - `tox` configuration file
+
+Subdirectories:
+* `myapp/myapp` - contains the Python source code and the unit test code. The directory can contain multiple subdirectories. Each directory might have a `tests` directory for the unit tests corresponding to the directory. `tests` directories might have `data` subdirectories containing the test data used in unit tests.
+
+
+### Development tools
+Contributions to the OpenCADC Python projects is expected to be submitted as Pull Requests (PR) from other forks of the project to the default branch (`main` or `master`).
```
Comment (SharonGoliath) [inline cadcutils/CONTRIBUTIONS.md:22]: projects *are* expected

File: cadcutils/CONTRIBUTIONS.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+### GitHub, Continuous Integration/ Continous Development (CI/CD)
+In order for a contribution (PR) to be merged into the project branch, it needs to be reviwed and accepted by at least one of the package maintainers and pass the CI/CD pre-conditions. These are implemented as GH Actions and check the following:
+- project `egg` builds successfully
+- unit tests are successfully executed in all the supported configurations (Python versions)
+- no code style errors are present
+- test code coverage does not decrease
+
+CADC staff can also run integration tests. These are not always accessible to external collaborators because they require Authentication/Authorization (A&A).
+
+Once the a patch has been merged into the default branch it can, depending on the urgency and its content, be published to `PyPI` immediately as an updated version or it can be bunddled with other minor changes to be released later. This is at the discretion of the package maintainers.
```
Comment (SharonGoliath) [inline cadcutils/CONTRIBUTIONS.md:69]: Once *a* patch

**Review by SharonGoliath — APPROVED** [review]
Nice.

**Comment by andamian** [comment]
Tracked with the proper document now: https://github.com/opencadc/.github/pull/3

**Pull request closed without merging**